webclient/pages/pf/index.vue

```diff
@@ -17,49 +17,35 @@ import VueAuthenticate from 'vue-authenticate'
 import Axios from 'axios'
 
 Vue.use(VueAxios, Axios)
-Vue.use(VueAuthenticate, {
-  baseUrl: 'http://192.168.0.100', // Your API domain
 
-  providers: {
-    github: {
-      clientId: '0370b3eb38840b3129ca',
-      redirectUri: 'http://192.168.0.100/auth/callback', // Your client app URL
-    },
-  },
-})
 export default {
   components: {},
   data: function () {
     return {
-      isAuthenticated: this.$auth.isAuthenticated(),
+      isAuthenticated: false,
       access_token: null,
       response: null,
     }
   },
   methods: {
     auth: function (provider) {
-      if (this.$auth.isAuthenticated()) {
-        this.$auth.logout()
+      const data = {
+        client_id: 'superstars-discourseonhttp',
```

Comment:
you can use the superstar ID.

But next time do not submit code without testing thast it work ?